packages/vitest/src/node/stdin.ts

```diff
@@ -5,7 +5,8 @@ import { isWindows, stdout } from '../utils'
 import type { Vitest } from './core'
 
 const keys = [
-  ['a', 'rerun all tests'],
+  [['a, return'], 'rerun all tests'],
```

Comment:
This breaks test run cancelling when pressing `'a'` as `cancelKeys` no longer contains `'a'`:

https://github.com/vitest-dev/vitest/blob/77d071a91f81b6faef28cb30c6c5b617a661459d/packages/vitest/src/node/stdin.ts#L15

Reply:
Thanks!!